1209/BABA/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: pid
+title: Famicom Cartridge Dumper/Writer
+owner: Cluster
+license: GPLv3
```

Comment:
Repo says MIT. Please match it.

```suggestion
license: MIT
```

Reply:
Actually I want to use it with totally new version based on STM32 and CPLD: https://github.com/ClusterM/famicom-dumper-writer
It's using GPLv3 license. I updated index.md.